data.js

```diff
@@ -0,0 +1,136 @@
+    href: "/info/caveat",
+    color: "blue",
+    title:
+      "Front-End, Back-End and support for the website of the creative studio Caveat",
+    url: "hellocaveat.com",
+    address: "New York, US",
+    header: "Website maintenance of creative studio Caveat",
+    stackTitle: "Stack: ",
+    stack:
+      "KeystoneJS, MongoDB, React, Typescript, GraphQL, Webpack, SCSS",
+    description:
+      ["Caveat is a creative agency based in New York. The agency is engaged in creative, copywriting, web , graphic and industrial designs. Caveat is growing and developing rapidly and the website needed to be supported and updated.","We were hired to give advice on optimizing the website infrastructure(architecture, Front-End, Back-End, deploy)"],
+    processHeader: "What we have done",
+    processDescription: ["По просьбе Кавеат мы подготовили подробный план и список необходимых доработок на стороне CMS и клиенте, включающий “минимум” — который закрывает базовые потребности, и “максимум” — отражающий все требования заказчика.", "В результате, мы структурировали и доработали CMS: добавили новые разделы, завели новые сущности, произвели миграцию старых полей в новые. На клиенте сверстали новые страницы в соответствии с макетами. Настроили процесс деплоя в связках github → heroku и github → netlify."]
+    
+  },
+  veneer: {
+    name: "Veneer",
+    images: {
+      large: "/images/5_Veneer.png",
+    },
+    logo: "/images/Veneer_White.svg",
+    href: "/info/veneer",
+    color: "pink",
+    title:
+      "Front-End development and support of the project management system Veneer",
+    url: "helloveneer.com",
+    address: "New York, US",
+    header: "Website maintenance of creative studio Veneer",
+    stackTitle: "Stack: ",
+    stack:
+      "Redux, Redux-Saga, Webpack, SCSS, Jest, React, Mobx, Fat client",
+    description:
+      ["Veneer — это онлайн сервис который помогает работникам креативной индустрии вести проекты: отслеживать прогресс, вовремя вносить правки. Цель проекта — сокращение потерь при производстве креатива.", "Мы подключились к проекту на стадии, когда у заказчика уже был готов дизайн 80% макетов."],
+    processHeader: "What we have done",
+    processDescription: ["Мы организовали прозрачный процесс доставки новых версий приложения до конечных пользователей. Разработали фронтенд-части с учетом потребностей заказчика. Выстроили прозрачные отношения между фронтенд и бекенд-разработчиками, что позволило ускорить разработку новых фичей.", "Наша экспертность помогла реализовать все задумки дизайнера и проектировщика интерфейса точно в  оговоренный срок а также решить многие потенциальные проблемы еще до их появления а также сделать работу в сервисе для пользователей максимально комфортной."]
+  },
+  MasterDelivery: {
+    name: "MasterDelivery",
+    images: {
+      large: "/images/4_MasterDelivery.png",
+    },
+    logo: "/images/Master_deliver_white.svg",
+    href: "/info/MasterDelivery",
```

Comment:
therefore, should fix too
seems it has a duplicate info (object key == href)
and string part "info" - also static, we actually remove this property (href) and just go use "object key"
it's okay practice, because we can't do mistake in two lines